### ISSUE TITLE
Missing steps/ information in classic_infra_keys.md

### ISSUE THREAD

**Issue by Nadine2016** [IBM-Bluemix-Docs/iam#27]
I replaced our existing steps for how to retrieve the infrastructure user name and API key as I was told to link to the classic_infra_keys.md page. But the page is very hard to follow and information is kind of hidden in the short description. 

These are the areas that would be great to improve: 

- there are no steps, some of the information regarding what permissions you need is hidden in the shortdesc that really should be in an important note or general note
- it is not clear that the account owner cannot see and create the API key for their users > this also feels like a limitation when a users loses their login credentials
- there are no steps for how to retrieve the infrastructure user name
- what is an ancestor in the classic infrastructure hierarchy?

**Comment by mekaufman** [comment]
@Nadine2016 I've made some updates internally to the topic to help address your feedback. Once it's reviewed, I will push it live. Thanks for the feedback.
